tests/protocol/test_connectors.py

```diff
@@ -434,9 +437,9 @@ async def test_connector_error():
 
 
 def mock_job(
-    status=JobStatus.COMPLETED,
-    error=None,
-    terminated=True,
+        status=JobStatus.COMPLETED,
+        error=None,
+        terminated=True,
```

Comment:
Do these indentations come from `make autoformat`?

Reply:
I think so. But I ran `make lint` in `main` and it didn't complain anything.

I think I will just revert all the changes.